javascript/javascript2/week3/The-fastest-presser-in-this-realm.js

```diff
@@ -0,0 +1,81 @@
+"use strict"
+
+const userInput = document.getElementById('number');
+const button = document.getElementById('button');
+const gameInfo = document.getElementById('game_info');
+const restart = document.getElementById('restart_button');
+
+function gameEnd() {
+  document.removeEventListener("keypress", keyPresses);
+  if (countOne > countTwo) {
+    gameInfo.innerHTML = "WINNER : S-key user";
+    var confettiSettings = {
+      target: 'my-canvas-one'
+    };
+    var confetti = new ConfettiGenerator(confettiSettings);
+    confetti.render();
+  } else if (countOne < countTwo) {
+    gameInfo.innerHTML = "WINNER: L-key user ";
+    var confettiSettings = {
+      target: 'my-canvas-two'
+    };
+    var confetti = new ConfettiGenerator(confettiSettings);
+    confetti.render();
+  } else if (countS === 0 && countL === 0) {
+    gameInfo.innerHTML = "No key presses!Try again";
+  } else if (countS === countL) {
+    gameInfo.innerHTML = "Its a Tie ";
+    console.log('Its a Tie');
+
```

Comment:
I think you are confused between `countOne,  countTwo ` and `countS, countL`.  `countOne,  countTwo ` have global scope with press count. `countS, countL` have local scope within the keypress function.  Try to fix game draw and no key press condition.